CITATION.cff

```diff
@@ -0,0 +1,34 @@
+cff-version: 1.2.0
+message: "Cite this paper whenever you use Bloqade"
+
+# authors:
+# - family-names: "John"
+#   given-names: "Doe"
+
+title: "Bloqade.jl: Package for the quantum computation and quantum simulation based on the neutral-atom architecture."
+version: "v1"
+license: "Apache License 2.0"
+# doi: "10.22331/q-2020-10-11-341"
+date-released: 2023-01-09
+url: "https://github.com/QuEraComputing/Bloqade.jl"
+# NOTE: add the reference paper here in the future
```

Comment:
Remove the commented lines below?

Reply:
I meant lines 14-34 here. Looks like it's copied from Yao. Should we remove those lines?